### ISSUE TITLE
dropout rate in train and predict

### ISSUE THREAD

**Issue by xudejing** [hx173149/C3D-tensorflow#16]
I find that in [train_c3d_ucf101.py](https://github.com/hx173149/C3D-tensorflow/blob/master/train_c3d_ucf101.py#L176) dropout is 0.5, while in [predict_c3d_ucf101.py](https://github.com/hx173149/C3D-tensorflow/blob/master/predict_c3d_ucf101.py#L108) dropout is 0.6, can you explain why ?  @hx173149

**Comment by hx173149** [comment]
Hi @xudejing 
I have try this value from 0.5 to 1.0 in predict process, and it can not make a meaningful difference, so 0.6 maybe just a debugging value

**Comment by xudejing** [comment]
I think during prediction, the dropout rate need to be 1, otherwise the activations are changed for layers applying dropout every time since dropout has a probability. if we use the model as a feature extractor., it is unstable.

**Comment by hx173149** [comment]
你好
是的，如果是拿来做特征提取用1.0确实会比较稳定，但是如果直接是用来做end to end的输出，在我测试的收 0.5-1.0 的效果都差不多。。。。

**Comment by xudejing** [comment]
好的~

**Comment by allenwangcheng** [comment]
看到中文好亲切
